spec/webmock/pulse_spec.rb

```diff
@@ -88,7 +88,12 @@
 
     end
 
-  end
+    it 'Creates pulse' do
+      stub_request(:post, "http://localhost:3030/api/pulse")
+        .to_return(status: 200, body: "OK")
 
+      expect(client.create_pulse).to have_requested(:post, "#{host}/api/pulse")
```

Comment:
https://github.com/metabase/metabase/blob/master/docs/api/pulse.md#post-apipulse mentioned request parameters for create pulse for example name, card, channel
test create pulse with correct request and response params